x/group/msgs_test.go

```diff
@@ -146,13 +146,6 @@ func TestMsgCreateGroupAccount(t *testing.T) {
 			timeout:   proto.Duration{Seconds: 1},
 			expErr:    true,
 		},
-		"valid admin required": {
-			admin:     []byte("invalid-address"),
```

Comment:
why is this test deleted?

Reply:
Now we have a variable-length address, this test never fails.